### PR TITLE
fix: Navbar2 to NabvaNews

### PR DESCRIPTION
**Pull request by Sstomi**

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ export default function Footer() {
                   src="/images/Home/zeteoh_kana_logo.png" // Route of the image file
                   height={39} // Desired size with correct aspect ratio
                   width={138} // Desired size with correct aspect ratio
-                  alt="zeteoh-logo"
+                  alt="zeteoh_logo"
```
Comment (dhassault) [inline components/Footer.js:105]: Why replacing `-` by `_`? Is it important as `alt`?

**Review by dhassault — APPROVED** [review]
Good work! I just have a question, just for curiosity. Anyway thanks!

**Review by dhassault — CHANGES_REQUESTED** [review]
[back-to-news-list](https://deploy-preview-22--zeteohai.netlify.app/news) is missing in the news post at the bottom.

File: components/Footer.js (hunk 1 above)
Comment (Sstomi) [reply components/Footer.js:105]: It's for google.com

**Review by dhassault — APPROVED** [review]

**Pull request merged**